### DISCUSSION TITLE
Add progress bar

### DISCUSSION THREAD

**Discussion by DinhHuy2010** [ThioJoe/YT-Spammer-Purge#705] [Ideas]
Can you add progress bar in downloading profile picture from JSON log?

**Comment by ThioJoe** [comment]
yea i'll add that to the to do list for the next version. Though not sure how long it will take, 2.17 isn't a huge priority because it's mostly "nice to have" features

**Reply by DinhHuy2010** [reply]
ok I got it

**Reply by Firecul** [reply]
> Ans also check this

Please don't take this as rudely as it probably sounds but why do we need to see you have tweeted about how many comments you are reporting?

**Reply by DinhHuy2010** [reply]
> > Ans also check this
> 
> Please don't take this as rudely as it probably sounds but why do we need to see you have tweeted about how many comments you are reporting?

ok fixing
